modules/test/playwright/tests/layout-set-prototype-web/siteTemplatesWithWebContent.spec.ts

```diff
@@ -56,6 +60,125 @@ const webContentName2: string = getRandomString();
 const webContentText1: string = getRandomString();
 const webContentText2: string = getRandomString();
 
+test('LPD-21445', async ({
+	apiHelpers,
+	applicationsMenuPage,
+	journalPage,
+	layoutSetPrototypePage,
+	page,
+	pageEditorPage,
+	pagesAdminPage,
+	productMenuPage,
+	sitesPage,
+	systemSettingsPage,
+	uiElementsPage,
+}) => {
+	const siteTemplateName: string = getRandomString();
+	const siteName1: string = getRandomString();
+	const siteName2: string = getRandomString();
+	const webContentName: string = getRandomString();
+	const text: string = getRandomString();
+	const secondPageNameOnSiteTemplate = getRandomString();
+
+	let site1Id: string | undefined;
+	let site2Id: string | undefined;
+
+	try {
```

Comment:
the try - finally block hides the actual test error if there is one (when during the test flow a step fails it goes directly to the final action, and in this context tries to delete a site that has not been even created yet)